web/frontend/src/components/slam-2d-render.vue

```diff
@@ -78,6 +114,24 @@ const disposeScene = () => {
   scene.clear();
 };
 
+/*
+ * Find the desired color bucket for a given probability. This assumes the probability will be a value from 0 to 100
+ * ticket to add testing: https://viam.atlassian.net/browse/RSDK-2606
+ */
+const probToColorMapBucket = (normProb: number, numBuckets: number): number => {
+  const prob = Math.max(Math.min(100, normProb * 255), 0);
```

Comment:
Hmm @JohnN193 , for my understanding is it really a scale from 0 to 1, or is it 0 to 100/255?

Also, "normalized" seems to be its own concept in [BufferAttribute](https://threejs.org/docs/index.html#api/en/core/BufferAttribute) so it'd be nice to avoid a name could be confused with that... could we simply call it 'probability'?

[note to self] carto_grpc_server sends back a value from 0-100, then [PCDLoader divides by 255](https://github.com/mrdoob/three.js/blob/0fbae6f682f6e13dd9eb8acde02e4f50c0b73935/examples/jsm/loaders/PCDLoader.js#L338-L340). We multiply by 255 here to bring it back to 0-100.

Reply:
Yeah for our implementation of a PCD from cartographer, normProb goes from 0-100/255. Because that isn’t a range I can guarantee for all slam algos I added the max/min bounds.

Fine with calling it probability. I picked the name just because I saw it was normalized but I don’t feel strongly about it

Reply:
Cool, calling it probability sounds good to me. Free tor resolve from my side.